lib/charms/layer/jenkins/api.py

```diff
@@ -1,4 +1,5 @@
 import requests
+from urllib.parse import urlparse
```

Comment:
Please follow [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) conventions, placing absolute imports before relative imports, and standard library imports before third-party and local library imports (all groups separated by a blank line):

```python
import requests

import jenkins

from urllib.parse import urlparse

from charmhelpers.core.hookenv import ERROR

from charms.layer.jenkins.credentials import Credentials
```